index.html

```diff
@@ -22,169 +22,184 @@ <h3 class="portfolio-nav--logo"><a href="#">My Logo</a></h3>
           <li><a href="#">Contact</a></li>
```

Comment:
[Optional]

- [ ] Awesome job with the website it looks great 👏 You missed only a small detail, kindly add the `id`s of the sections of the `href` attributes of the navbar links to make them functional 👍

Reply:
Thanks for your honest review, @ha-manel.